qa/workunits/ceph-helpers.sh

```diff
@@ -513,6 +524,9 @@ function destroy_osd() {
     local dir=$1
     local id=$2
 
+    # First mask osd down, otherwise it cannot be deleted later on
+    # is status has not propagated to monitor.
```

Comment:
could you elaborate this a little bit? why we need to mark this osd down explicitly? and if we really need to do this. maybe you can move it down to line 531. just for consistency.